app/controllers/api/v2/resolved_urls_controller.rb

```diff
@@ -0,0 +1,31 @@
+# frozen_string_literal: true
+
+class Api::V2::ResolvedUrlsController < Api::BaseController
```

Comment:
Any reason to make this v2? There is no v1 for this API?

Reply:
No reason, just that I had not fully internalized our policy here.